.babelrc

```diff
@@ -1,6 +1,15 @@
 {
-  "stage": 1,
-  "optional": ["runtime"],
-  "loose": ["all"],
-  "plugins": ["dev-expression"]
+  "presets": [
+    "react",
+    "es2015",
+    "stage-1"
+  ],
+
+  "plugins": [
+    "transform-runtime",
+    "add-module-exports",
+    "transform-decorators-legacy",
```

Comment:
I think I'd prefer to just refactor the two places we use decorators to not use them, lowers complexity and makes it more consistent with the rest of the codebase anyway